feedhenry/mcp-standalone
# ✨ WIP: FH-4023 Import the UI extension code & installation setup

* add the `ui` folder from https://github.com/feedhenry/mobile-control-panel
* resovle the MCP Route for each mcp controller
* add the install script & playbook config for installing the mcp UI extension into OpenShift


To verify:
```bash
cd installer
ansible-playbook ./playbook.yml -e "dockerhub_username=<username>" -e "dockerhub_password=<password"" --ask-become-pass
```

Then navigate to the Service catalog in OpenShift and provision an 'mcp' into your project.
Once provisioned, visit the 'Mobile' left nav item in that Project. 
You should now be able to create 'Mobile Apps'.
The Mobile App  view should also work.
If a Mobile Enabled Service is provisioned (e.g. fh-sync-server), it should show under the 'Provisioned Services' section in the Mobile left nav view.
